components/Button/Button.tsx

```diff
@@ -38,7 +38,7 @@ const buttonVariants = cva(
 export interface ButtonProps
   extends React.ButtonHTMLAttributes<HTMLButtonElement>,
     VariantProps<typeof buttonVariants> {
-  label?: React.ReactNode;
+  label?: boolean | JSX.Element | string;
   icon?: JSX.Element;
```

Comment:
I'd love to know more about your thoughts as to why label has the possibility of 3 data types. :) Please let me know.

Reply:
@vmaineng I set it up that way as it would be `true` if it is defined, a string directly passed to the button like "Continue", or the JSX.Element , which in this case, is the LoadingSpinner :)